### PR TITLE
update activity log function

### PR DESCRIPTION
**Pull request by jesperskjonnemand**
By following and to some extend understanding the instructions on this brilliant tutorial I managed to create a friendly welcome screen for my editors. The dashboard includes the suggested activity log which is a nice feature.

However, if editors save or publish a page several times, it will appear on the "pick up where you left off" list several times. Ideally, a node would only appear on the list once, regardless of multiple saves or publishes.

The activity log function seems to attempt to take care of this, but for some reason it does not, at least not in my case. 

So being a Angular JS rookie I tried to rewrite the loop to better understand what is going on. And this code actually works. Maybe other users could benefit from this.

### CODE REVIEW

**Comment by eshanrnh** [comment]
Thanks for the PR, @jesperskjonnemand 🙌 We will review it asap.

File: Tutorials/Creating-a-Custom-Dashboard/index.md [hunk 1]
```diff
@@ -544,50 +563,73 @@ angular.module("umbraco").controller("CustomWelcomeDashboardController", functio
...
             console.log(response);
             vm.UserLogHistory = response;
+
+            // define the entity types that we care about, in this case only content and media
+            var supportedEntityTypes = ["Document", "Media"];
+
+            // define an empty array "nodes we know about"
+            var nodesWeKnowAbout = [];
+
+            // definde an empty array "filtered log entries"
```
Comment (sophie-neale) [inline Tutorials/Creating-a-Custom-Dashboard/index.md:578]:
```suggestion
            // define an empty array "filtered log entries"
```

**Review by sophie-neale — COMMENTED** [review]
Spotted a couple sneaky typos so suggested fixes for you @jesperskjonnemand ☺️

File: Tutorials/Creating-a-Custom-Dashboard/index.md (hunk 1 above)
Comment (jesper-skjonnemand) [reply Tutorials/Creating-a-Custom-Dashboard/index.md:578]: Hi @sophie-neale. Well spotted :-) I am pretty new to this so I am not sure if I am supposed to do anything now to make the corrections happen (?)

Comment (eshanrnh) [reply Tutorials/Creating-a-Custom-Dashboard/index.md:578]: Hey @jesperskjonnemand,

To apply the changes/corrections, click on "Commit suggestion" -> "Commit changes".

**Review by marcemarc — APPROVED** [review]

**Comment by sofietoft** [comment]
Hi @jesperskjonnemand !

Thanks again for updating this tutorial with some great code - I'm especially fond of all the comments 🤩 
Looks like we're ready to merge it in 💪 

I see that this is your first ever contribution to the UmbracoDocs - that's **amazing**! #H5YR 
Do you have an account on Our?
If you do, I can make sure you get a C-trib badge, so everyone can see that you're now officially a contributor 😎 

Looking forward to see you back here with more awesome stuff!

**Comment by jesper-skjonnemand** [comment]
Hi Sofie

Happy to hear that the updated code is useable.

Comments are a great tool for a rookie like me -- I like to start any task by writing a bunch of comments describing the steps I want to take in plain words and then go ahead and try to formulate this in the 'local language'.

My Our-profile is here https://our.umbraco.com/members/id:276049/. The C-trib badge would of course make me incredibly proud 🙂

Best wiskes
Jesper
________________________________

**Comment by sofietoft** [comment]
That sounds like a great approach 👏 😄 

You now have a C-trib badge to flash on your Our profile!
Thanks again for helping out with our docs - would be awesome to see you back here again 😎 

Cheers!

**Pull request merged**